CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [unreleased]
+## [2.1.0] - 2020-08-25
 
 ### Added
```

Comment:
I think it worth mentioning the newly added support for the BooleanAttributeInputCallback Support and NumberAttributeInputCallback Support. 
This would be consistent with https://sdks.forgerock.com/getting-started/compatibility/#callbacks

I know that they are part of the "IDM nodes"... up to you...

Reply:
done